packages/web/package.json

```diff
@@ -7,12 +7,12 @@
   "private": true,
   "scripts": {
     "publish-scripts": "./scripts/publishScripts.sh",
-    "start": "craco --max-old-space-size=8076 start",
+    "start": "craco --max-old-space-size=12288 start",
```

Comment:
Why this bump?

Reply:
idk! i oomed today randomly. :( maybe not needed?